nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/resources/csv/multi-bank-account_escapedchar.csv

```diff
@@ -0,0 +1,3 @@
+idnamebalanceaddresscitystatezipCodecountry
```

Comment:
All files need an Apache 2.0 License header, or if that is not feasible due to the nature/format of the file, it must be added to the Apache Rat exclusion list in the nifi-record-serialization-services POM.

Reply:
Sorry. My bad. I ran an entire contrib check skipping rat checks. Added the new CSV to the exclusion list.

Did that with rat enabled for this bundle alone (for some reason rat checks fail at the parent level level - for nifi-commons).